test/allow_list.js

```diff
@@ -0,0 +1,280 @@
+        numAllowedAccounts = await allowListContract.numAllowedAccounts();
+        assert.equal(numAllowedAccounts, 2);
+    });
+
+    it("can be removed from allow list, and activated validators are removed", async () => {
+        await addValidators(1, 1);
+
+        let currentValidators = await allowListContract.getValidators();
+        assert.lengthOf(currentValidators, 2);
+        assert.equal(currentValidators[0], validators[0]);
+        assert.equal(currentValidators[1], validators[1]);
+
+        await allowListContract.voteToRemoveAccountFromAllowList(accounts[0], {from: accounts[0]});
+        await allowListContract.voteToRemoveAccountFromAllowList(accounts[0], {from: accounts[1]});
+        await allowListContract.countVotes(accounts[0]);
+        numAllowedAccounts = await allowListContract.numAllowedAccounts();
+        assert.equal(numAllowedAccounts, 1);
+
+        currentValidators = await allowListContract.getValidators();
+        assert.lengthOf(currentValidators, 1);
+        assert.equal(currentValidators[0], validators[1]);
+    });
+
+    it("can remove a vote", async () => {
+        await addValidators(1, 2);
+
+        let numAllowedAccounts = await allowListContract.numAllowedAccounts();
+        assert.equal(numAllowedAccounts, 3);
+
+        await allowListContract.voteToAddAccountToAllowList(accounts[3], {from: accounts[0]});
+        await allowListContract.voteToAddAccountToAllowList(accounts[3], {from: accounts[1]});
+
+        await allowListContract.removeVoteForAccount(accounts[3], {from: accounts[0]});
+
+        await allowListContract.countVotes(accounts[2]);
+        numAllowedAccounts = await allowListContract.numAllowedAccounts();
+        assert.equal(numAllowedAccounts, 3);
+    });
+
+    it("can remove a vote", async () => {
+        await addValidators(1, 2);
+
+        let numAllowedAccounts = await allowListContract.numAllowedAccounts();
+        assert.equal(numAllowedAccounts, 3);
+
+        await allowListContract.voteToAddAccountToAllowList(accounts[3], {from: accounts[0]});
+        await allowListContract.voteToAddAccountToAllowList(accounts[3], {from: accounts[1]});
+
+        await allowListContract.removeVoteForAccount(accounts[3], {from: accounts[0]});
+
+        await allowListContract.countVotes(accounts[2]);
+        numAllowedAccounts = await allowListContract.numAllowedAccounts();
+        assert.equal(numAllowedAccounts, 3);
+    });
+
+    it("can call countVotes to get number of votes for an account", async () => {
+        await addValidators(1, 1);
+
+        await allowListContract.voteToAddAccountToAllowList(accounts[2], {from: accounts[0]});
+        await allowListContract.voteToAddAccountToAllowList(accounts[2], {from: accounts[1]});
+        const web3Contract = new web3.eth.Contract(allowListContractAbi, allowListContract.address);
+        let numVotes = await web3Contract.methods.countVotes(accounts[2]).call({from: accounts[0]});
```

Comment:
Why don't you use `allowListContract` here and create a fresh `web3.eth.Contract` wrapper instead?